sktime/classification/deep_learning/base.py

```diff
@@ -66,6 +66,18 @@ def build_model(self, input_shape, n_classes, **kwargs):
         """
         ...
 
+    def get_metrics(self):
```

Comment:
could you call this `summary` to be in line with #971 and `statsmodels` type models?

Reply:
Done